dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -91,12 +91,15 @@ public static Path generate()
     ArtifactCache cache = loadArtifactInfo(managedDependencies);
     
     ImmutableList<Path> classpath = StaticLinkageChecker.artifactsToClasspath(managedDependencies);
```

Comment:
Do you still use classpath?

Reply:
yes, on line 97. I should get rid of this but my first attempt broke things.